homeassistant/components/ecobee/services.yaml

```diff
@@ -1,104 +1,222 @@
 create_vacation:
+  name: Create vacation
   description: >-
     Create a vacation on the selected thermostat. Note: start/end date and time must all be specified
     together for these parameters to have an effect. If start/end date and time are not specified, the
     vacation will start immediately and last 14 days (unless deleted earlier).
   fields:
     entity_id:
-      description: ecobee thermostat on which to create the vacation (required).
+      name: Entity
+      description: ecobee thermostat on which to create the vacation.
+      required: true
       example: "climate.kitchen"
+      selector:
+        entity:
+          integration: ecobee
+          domain: climate
     vacation_name:
-      description: Name of the vacation to create; must be unique on the thermostat (required).
+      name: Vacation name
+      description: Name of the vacation to create; must be unique on the thermostat.
+      required: true
       example: "Skiing"
+      selector:
+        text:
     cool_temp:
-      description: Cooling temperature during the vacation (required).
+      name: Cool temperature
+      description: Cooling temperature during the vacation.
+      required: true
       example: 23
+      selector:
+        number:
+          min: -23
+          max: 120
+          step: 0.5
```

Comment:
The ecobee integration does not specify a min/max temperature, thus inherits the default from the Climate entity component. Which is between 7 & 35 degrees celsius.

Step is 1.0 in this case I think?

Reply:
These ranges are the limits that ecobee allows its thermostats to be set to when considering both Fahrenheit and Celsius. Those who use Celsius can step the temperature at 0.5. Admittedly, the current range should be shortened. Please advise on the step.